diff --git a/libarchive/test/test_sparse_basic.c b/libarchive/test/test_sparse_basic.c
index 133a8539..61b0ff5d 100644
--- a/libarchive/test/test_sparse_basic.c
+++ b/libarchive/test/test_sparse_basic.c
@@ -53,7 +53,7 @@ __FBSDID("$FreeBSD$");
 /* The logic to compare sparse file data read from disk with the
  * specification is a little involved.  Set to 1 to have the progress
  * dumped. */
-#define DEBUG 0
+//#define DEBUG 0
 
 /*
  * NOTE: On FreeBSD and Solaris, this test needs ZFS.
